index.js

```diff
@@ -6,58 +6,30 @@ export default function pLimit(concurrency) {
 	const queue = new Queue();
 	let activeCount = 0;
 
-	const resumeNext = () => {
```

Comment:
The `next` after the change is almost the same as the `resumeNext` before the change.
It seems to me that the change is easier to understand if the unchanged part is unchanged.